make/upgrade

```diff
@@ -15,8 +15,7 @@ cd "${GIT_ROOT}"
 . make/include/secrets
 
 # Argument? (**)
-RELEASE=$(helm list --date --reverse --max 1 --namespace "${NAMESPACE}" -q | tail -n 1)
-echo Upgrading ${NAMESPACE} release \"${RELEASE}\" ...
+echo Upgrading ${NAMESPACE} release \"${NAMESPACE}\" ...
```

Comment:
This would be clearer, I think, if we do a `RELEASE="${NAMESPACE}" (with appropriate comment) instead?

Reply:
:+1: Fixed and rebased to simplify the diff.